### DISCUSSION TITLE
FEM contact buffer overflow detected, please increase its size in the scene desc!

### DISCUSSION THREAD

**Discussion by rossetti73** [NVIDIA-Omniverse/PhysX#260] [General]
hello!!
We face this problem during cloth simulation with softbody.

error is this :
**C:\b\w\eb2f45c4acc808a0\physx\source\gpusimulationcontroller\src\PxgFEMCore.cpp (196) : internal error : FEM contact buffer overflow detected, please increase its size in the scene desc!**

Some people how how to increase this buffer contact.

We search on scenedesc but not find any buffer .
Thanks in advance for any help. 
![image](https://github.com/NVIDIA-Omniverse/PhysX/assets/72785510/4b6265dc-ea69-4447-9473-97ac9fcc44f9)

**Comment by adenzler-nvidia** [comment]
Hi rossetti73

What version are you using? I agree that error message is very ambiguous. We have improved this on the latest development trunk and it will tell you exactly what parameter to change in the future.

The numbers are all part of PxgDynamicsMemoryConfig, which is defined in PxSceneDesc.h. In your specific case, the parameter that should be changed would be maxSoftBodyContacts.

**Comment by rossetti73** [comment]
Thanks @adenzler-nvidia !!!
very good solve problem !!!
`sceneDesc.gpuDynamicsConfig.maxSoftBodyContacts = (2 * 1024 * 1024),  //default 1 * 1024 * 1024 ; `
![image](https://github.com/NVIDIA-Omniverse/PhysX/assets/72785510/d77e4a9b-321f-4b31-bf23-09c1d6f3f4c5)
